diff --git a/dist/index.mjs b/dist/index.mjs
index 8ba42f0088393ba8d50abda6f79cc80466d48068..ed3c29ea2b57a4cbc28c1d553b81caaa228f191f 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -8,7 +8,7 @@ import { randomUUID } from "node:crypto";
 import { createServer, preview } from "vite";
 import { parseSync } from "oxc-parser";
 import MagicString from "magic-string";
-import { transformComponentsForBrowser, transformComponentsForServer } from "remix/ui-hmr";
+import { transformComponentsForBrowser, transformComponentsForServer } from "remix/component-hmr";
 //#region src/prerender-target.ts
 /**
 * Opens the built application for prerendering.
